README.md

```diff
@@ -140,17 +140,19 @@ from GitHub. You may make your own fork project from our project.
 ### 1. Debian/Ubuntu
 
 - gcc
-- libncurses-dev
-- libreadline-dev
 - make
-- openssl-dev
+- libreadline-dev
+- libssl-dev
+- libncurses-dev
+- zlib1g-dev
 
 ### 2. Redhat/CentOS
 - gcc
-- openssl-devel
 - make
-- ncurses-devel
 - readline-devel
+- openssl-devel
+- ncurses-devel

```

Comment:
List should be alphabetically sorted. And where is the dependancy on zlib for Redhat?

Reply:
This is your own. not changed. conference by others.if needed I will test.